save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/Plugin.kt

```diff
@@ -158,6 +159,28 @@ abstract class Plugin(
         }
     }
 
+    /**
+     * Substitute placeholders in `this.execFlags` with values from provided arguments
+     *
+     * @param execFlags a command that will be executed to check resources and emit warnings
+     * @param extraFlags [ExtraFlags] to be inserted into `execFlags`
+     * @param fileNames file name or names, that need to be inserted into `execFlags`
+     * @return `this.execFlags` with resolved placeholders
+     */
+    fun resolvePlaceholdersFrom(
+        execFlags: String?,
+        extraFlags: ExtraFlags,
+        fileNames: String): String =
+            execFlags!!
```

Comment:
Since we call `!!` anyway, we should either make argument nn-nullable, or use `requireNotNull` with error message here as a precondition